mjpc/tasks/humanoid/tracking/task.xml

```diff
@@ -137,14 +138,14 @@
     <framelinvel name="tracking_linvel[rhip]" objtype="site" objname="tracking[rhip]" />
   </sensor>
 
-  <!-- <include file="./keyframes/CMU-CMU-02-02_04_poses.xml" />  simple jump, balance -->
-  <!-- <include file="./keyframes/CMU-CMU-87-87_01_poses.xml" />  <!-\- jump with kick and spin -\-> -->
-  <!-- <include file="./keyframes/CMU-CMU-88-88_06_poses.xml" />  <!-\- jump and spin kick -\-> -->
-  <!-- <include file="./keyframes/CMU-CMU-88-88_07_poses.xml" />  <!-\- cartwheel-\-> -->
-  <!-- <include file="./keyframes/CMU-CMU-88-88_08_poses.xml" />  <!-\- crouch and flip backward on hands -\-> -->
-  <!-- <include file="./keyframes/CMU-CMU-88-88_09_poses.xml" />  <!-\- stretch and cartwheel -\-> -->
+  <include file="./keyframes/CMU-CMU-02-02_04_poses.xml" />  simple jump, balance
```

Comment:
The comment on this line is different from the ones below. Maybe change to `<!-\- simple jump, balance -\->` or remove `<!-\-` and `-\->` from the others

Reply:
Standardized the comment style.